server/initialize.coffee

```diff
@@ -85,11 +105,13 @@ initFiles = (callback) ->
             callback err
         else
             mkdirp config('dir_log'), (err) ->
-                if process.env.NODE_ENV is "production" or
-                        process.env.NODE_ENV is "test"
-                    initTokenFile callback
-                else
-                    callback()
+                mkdirp config('dir_app'), (err) ->
+                    initAppsDir (err) ->
+                        if process.env.NODE_ENV is "production" or
+                                process.env.NODE_ENV is "test"
```

Comment:
Maybe we can just test for `development`. For security thing, I prefer that unexpected value go to the more secure path. Here, if someone makes a typo like https://github.com/scaleway-community/scaleway-cozy/commit/678457054d64828c0d0f014e7ed268cf553fa87a, it goes to the development path.

```js
if process.env.NODE_ENV isnt "development"
```